sopel/plugins/handlers.py

```diff
@@ -330,3 +330,51 @@ def reload(self):
         module might not be available through ``sys.path``.
         """
         self._module = self._load()
+
+
+class EntrypointPluginHandler(PyModulePlugin):
+    """Sopel plugin loaded from a ``setuptools`` entry points
```

Comment:
Either "a" or "points", but not both. 😁

```suggestion
    """Sopel plugin loaded from a ``setuptools`` entry point
```